tools/src/code-review/reviewers/checkMissingChangelogs.ts

```diff
@@ -36,7 +36,8 @@ export default async function ({ pullRequest, diff }: ReviewInput): Promise<Revi
     title: 'Missing changelog entries',
     body: `Your changes should be noted in the changelog. 
       Read [Updating Changelogs](https://github.com/expo/expo/blob/main/guides/contributing/Updating%20Changelogs.md) 
-      guide and consider (it's optional) adding an appropriate entry to the following changelogs: ${changelogLinks}`,
+      guide and consider (it's optional) adding an appropriate entry to the following changelogs:
+${changelogLinks}`,
```

Comment:
This needs to have no indent before newline or it breaks 😕 Should I paste some `\n` manually? Like
```suggestion
      \n${changelogLinks}`,
```
or
```suggestion
      ${'\n' + changelogLinks}`,
```

Reply:
What you think about this approach since the break line is a required part of changelog block anyway? (sorry for the format, but I cannot comment on unmodified lines)
```ts
const changelogLinks = '\n' + pkgsWithoutChangelogChanges
```

Reply:
That's what I suggested too 😉 
> Last option is to prepend the changelogLinks variable with `\n` when it's created a few lines above

Reply:
Oh, I see  it now in the last sentence. 😅 Codeblocks have stolen my attention, sorry!

Reply:
Both the first and the last options seem fine to me 👍 Btw do you know what broke this? It was displayed well some time ago 🤔